android/apollo/src/com/andrew/apollo/MusicPlaybackService.java

```diff
@@ -1357,7 +1365,7 @@ private void notifyChange(final String what) {
         // Update the lockscreen controls
         updateRemoteControlClient(what);
 
-        if (what.equals(POSITION_CHANGED)) {
+        if (what.equals(POSITION_CHANGED) || what.equals(PLAYSTATE_STOPPED)) {
```

Comment:
This code tells me that it was a mistake using `notifyChange(PLAYSTATE_STOPPED)`.

If all you actually needed was to call `updateRemoteControlClient(PLAYSTATE_STOPPED)`